### ISSUE TITLE
Thank you for your work. You have only provided the refined annotations for the test set. Could you please provide the refined annotations for the training set for the sake of subsequent research and fair comparison?

### ISSUE THREAD

**Issue by ydhcg-BoBo** [Songinpyo/SFTrack#2]
Thank you for your work. You have only provided the refined annotations for the test set. Could you please provide the refined annotations for the training set for the sake of subsequent research and fair comparison?

**Comment by Songinpyo** [comment]
Thanks for your interest.

We only refined the annotations in test set of UAVDT dataset.

**Comment by ydhcg-BoBo** [comment]
Thank you. This is a great work, how long will it take to open source the model and weights?
